code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -513,16 +514,8 @@ func (uar *updateAllocationRequest) validate(
 // calculate size difference for every blobber of the allocations
 func (uar *updateAllocationRequest) getBlobbersSizeDiff(
```

Comment:
this can also be named `bSize()`
as there is no diff.. it's just getting how much to be extended/reduced to per blobber

Reply:
Yes we  should remove either getBlobbersSizeDiff or bsize as they are the same
```go
func (sa *StorageAllocation) bSize() int64 {
	var size = sa.DataShards
	return int64(math.Ceil(float64(sa.Size) / float64(size)))
}
```

Reply:
yeah that's why I said to change this name too to `bSize()`
updateAllocationRequest.bSize()
allocation.bSize()

anyhow the name `getBlobbersSizeDiff` doesn't sound correct.